src/app/children/aser/aser.ts

```diff
@@ -47,13 +37,19 @@ export class Aser extends Entity {
     'Division'
   ];
 
-
+  @dbProperty('string')
   child: string; // id of Child entity
+  @dbProperty('Date')
```

Comment:
Is it possible to use the TypeScript typings of the annotated class attributes implicitly (i.e. instead of writing 'Date' or 'string' explicitly in the annotation, infer that from `date: Date`)?

Reply:
You can retrieve the type by using the `reflact-metadata` API, but to get the more sophisticated information like to determine if it is an option parameter is not supported yet (one aproach can be found [here](https://stackoverflow.com/a/34701754/10713841)). But I think that is way to "hacky". So should we just retrieve the type information automatically or just leave it as it is with the string regex?